docs/bgp-policy.md

```diff
@@ -81,8 +81,12 @@ as the effective BGPPolicy.
 
 ### LocalASN
 
-The `localASN` field defines the Autonomous System Number (ASN) that the local BGP process uses. The available private
-ASN range is `64512-65535`. The field is mandatory.
+The `localASN` field defines the Autonomous System Number (ASN) that the local BGP process uses. This field is mandatory
+and accepts values in the range of `1-4294967295`.
+
+For private ASNs, which are within the ranges 64512-65534 (16-bit) or 4200000000-4294967294 (32-bit), should be strictly
+limited to private networks or environments that do not peer with public ASNs. If public network connectivity is required,
+coordinate with your upstream provider to avoid issues caused by private ASN usage.
```

Comment:
So BGP process on Antrea will advertise the routes regardless of the ASN type? Not sure if it's expected to do so or we should do anything to prevent private ASN routes to be public?

Reply:
It's not our obligation to prevent users from peering private ASN to public BGP ASs. To some extent, we should assume that users have the knowledge about that, similar to RFC 1918 defining the private ranges of IP. To warn users about the private ASNs, we add some explanations here.